### DISCUSSION TITLE
docs/courses/cb&b%20634/2024-10-22/

### DISCUSSION THREAD

**Discussion by giscus** [open-genome/open-genome.github.io#12] [Ideas]
# docs/courses/cb&b%20634/2024-10-22/

A starter template for a Jeykll site using the Just the Docs theme!

https://open-genome.github.io/docs/courses/cb&b%20634/2024-10-22/

**Comment by super-dainiu** [comment]
good note

**Comment by GabbyKoki** [comment]
good note
